app/components/embed/media_component.html.erb

```diff
@@ -28,7 +28,7 @@
           </select>
         </div>
       </div>
-      <div class="transcript" data-action="media-loaded@window->transcript#persistPlayer time-update@window->transcript#highlightCue">
+      <div class="transcript" data-action="media-loaded@window->transcript#persistPlayer media-data-loaded@window->transcript#load time-update@window->transcript#highlightCue">
```

Comment:
media-data-loaded is a custom event that fires when the first track has a "loadeddata" event.  When this event fires, the transcript load function is executed